addons/web/static/src/views/form/form_renderer.js

```diff
@@ -42,7 +42,14 @@ export class FormRenderer extends Component {
         });
         this.uiService = useService("ui");
         this.onResize = useDebounced(this.render, 200);
-        onMounted(() => browser.addEventListener("resize", this.onResize));
+        onMounted(() => {
+            browser.addEventListener("resize", this.onResize)
+
+            const aliasEl = document.querySelector('div[name="alias_def"]');
+            if (aliasEl) {
+                aliasEl.style.setProperty('direction',"ltr")
+            }
+        });
```

Comment:
Hello. I don't think this kind of stuff should be handled by the framework. You can simply set attribute `dir="ltr"` on `alias_def` divs, in the archs.

Reply:
> Hello. I don't think this kind of stuff should be handled by the framework. You can simply set attribute `dir="ltr"` on `alias_def` divs, in the archs.

Hello @aab-odoo , it appears that the issue persists with every alias_def (domain picker), and in some instances, it is referenced through XPath. It seems like a more efficient approach would be to address it comprehensively if it's fine? Thankyou :)

Reply:
I don't think it is. `alias_def` isn't any sort of framework element. It's something that has been copy pasted from views to views over time, so having a special case for this in the framework makes no sense. AFAICS there're 10 occurrences in the codebase. There's a native html attribute to achieve what you want, there's no need to introduce a layer on top of it in the framework.  The simpler approach is to add the `dir` attribute on each of them. If the fix was more complicated that this, I would suggest to develop a widget, which would be the adequate way to add the support for this in the framework. But here, we're talking about a "9 letter"-fix, so the simpler the better, imo.

Reply:
> I don't think it is. `alias_def` isn't any sort of framework element. It's something that has been copy pasted from views to views over time, so having a special case for this in the framework makes no sense. AFAICS there're 10 occurrences in the codebase. There's a native html attribute to achieve what you want, there's no need to introduce a layer on top of it in the framework. The simpler approach is to add the `dir` attribute on each of them. If the fix was more complicated that this, I would suggest to develop a widget, which would be the adequate way to add the support for this in the framework. But here, we're talking about a "9 letter"-fix, so the simpler the better, imo.

Oh okay got it, thanks for clarifying! We'll proceed with the suggested approach then. :)